doc/classes/AnimatedSprite.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AnimatedSprite" inherits="Node2D" version="4.0">
 	<brief_description>
-		Sprite node that can use multiple textures for animation.
+		Sprite node that contains multiple textures as frames to play for animation.
 	</brief_description>
 	<description>
-		Animations are created using a [SpriteFrames] resource, which can be configured in the editor via the SpriteFrames panel.
+		This node is similar to the Sprite node, except instead of the [TextureRegion] resource in the Bottom Panel it has the [SpriteFrames] resource. Animations are created using a [SpriteFrames] resource, where it allows you to import image files (or a folder containing said files) to provide the animation frames for the sprite. The [SpriteFrames] resource can be configured in the editor via the SpriteFrames panel.
```

Comment:
This is incorrect and misleading. TextureRegion is not a resource, it's not even a class (you can try clicking it in the doc). It's an editor for editing the texture region property of the Sprite. It's completely unrelated to SpriteFrames.